NitroxServer/Communication/Packets/Processors/ServerCommandProcessor.cs

```diff
@@ -7,7 +7,13 @@ public class ServerCommandProcessor : AuthenticatedPacketProcessor<ServerCommand
     {
         public override void Process(ServerCommand packet, Player player)
         {
+#if Debug_WPF
+           MainWindow._Server.Save(); 
+#endif
+#if Debug
```

Comment:
What about release?

Is there a way to pass in an instance of server instead of accessing these statically?